main.go

```diff
@@ -146,6 +150,17 @@ func run() error {
 			return xerrors.Errorf("error in Photon update: %w", err)
 		}
 		commitMsg = "Photon Security Advisories"
+	case "ghsa":
+		src := oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: githubToken},
```

Comment:
I'm assuming this GitHub token will actually require the same level of permissions that a token to access the GraphQL api would?

Reply:
graphql api does not require any scope...
Should I use another access token?